test/classification/training/test_trainer.py

```diff
@@ -216,6 +216,35 @@ def test_warmup(self):
         trainer.fit(model, [dataloaders[0]])
         self.assertEqual(trainer.warmup_steps, 1)
 
+    def test_save_load(self):
```

Comment:
have you tested this with resuming training for a saved checkpoint?

Reply:
yes, I have included it in the test now